src/vm/perfmap.cpp

```diff
@@ -51,9 +51,20 @@ void PerfMap::Initialize()
         {
             s_ShowOptimizationTiers = true;
         }
-#ifndef CROSSGEN_COMPILE
-        PerfJitDump::Start();
-#endif
+
+        char jitdumpPath[4096];
+
+        // CLRConfig::GetConfigValue(CLRConfig::EXTERNAL_PerfMapJitDumpPath) returns a LPWSTR
+        // Use GetEnvironmentVariableA because it is simpler.
+        // Keep comment here to make it searchable.
+        auto len = GetEnvironmentVariableA("COMPlus_PerfMapJitDumpPath", jitdumpPath, sizeof(jitdumpPath) - 1);
```

Comment:
A nit - can you please use a specific type instead of `auto`? The convention is to use auto only in places where the type cannot be expressed otherwise, like for lambdas.